.circleci/config.yml

```diff
@@ -297,6 +300,22 @@ jobs:
             else
               DRY_RUN=true python3 .circleci/issue_tracker.py
             fi
+  solidity-checking:
```

Comment:
I think you can add a `- working_directory: ./crypto/stark-verifier-ethereum` to avoid all the `cd`'s later: https://circleci.com/docs/2.0/configuration-reference/#job_name

Reply:
May need to be `~/project/crypto/stark-verifier-ethereum`

Reply:
Beautiful will do